### DISCUSSION TITLE
Some q&a

### DISCUSSION THREAD

**Discussion by MrLololoshkaSKINHEAD7777PRO334566SIGMA** [Fedoraware/Fedoraware#1167] [Q&A] [answered]
Hello guys, i have some q&a for cheat, 1) how it play's on 2023, 2) is fedora detect?, 3) are fedora updaiting and where install the last realese?.

**Comment by HutchyBen** [comment, marked as answer]
1. yes
2.  probably
3. very updating use fedoraloader to inject up to date and apply vac bypass
